README.md

```diff
@@ -1 +1,50 @@
-# ramen
+# Ramen
+
+Ramen is a **disaster-recovery orchestrator** for stateful applications across
+a set of peer kubernetes clusters that,
+
+- Preserve state using persistent volumes provisioned using [ceph-csi](https://github.com/ceph/ceph-csi/)
+- Uses the CSI [storage replication extensions](https://github.com/csi-addons/volume-replication-operator)
```

Comment:
"Uses the CSI Addons" instead of "Uses the CSI"?

The first bullet seems to be a functionality (what), the second bullet seems to describe (how) the functionality is achieved.  Hence, should the two be combined or the second be made a sub-bullet of the first?  We should add another bullet on the lines of "Preserves PV metadata for use across the peer clusters in a DR relationship" to show the two aspects of "state": data and metadata.

Reply:
Ack on the former, and not stating the latter (preserving PV metadata) explicitly. This is a function of VolRep, and leaving it out of this introduction. It will be added in other places like install (or sublinks) where we deal with S3.

Reply:
I was referring to the function of storing PV resource in S3 store, which is not a VolRep function.

Reply:
Ah yes, what I meant was preserving PV and reattaching it is how VolRep works. If in the future we preserve a cookie instead that VolRep can act on, instead of the PV we would mention that or just say state metadata.

As it stands we are not saying "how" it works in the README, so we can add these details in the other docs?